build/common.sh

```diff
@@ -582,26 +582,13 @@ function kube::build::run_build_command_ex() {
   fi
 }
 
-function kube::build::probe_address {
-  # Apple has an ancient version of netcat with custom timeout flags.  This is
-  # the best way I (jbeda) could find to test for that.
-  local netcat
-  if nc 2>&1 | grep -e 'apple' >/dev/null ; then
-    netcat="nc -G 1"
-  else
-    netcat="nc -w 1"
-  fi
-
+function kube::build::rsync_probe {
   # Wait unil rsync is up and running.
-  if ! which nc >/dev/null ; then
-    V=6 kube::log::info "netcat not installed, waiting for 1s"
-    sleep 1
-    return 0
-  fi
-
-  local tries=10
+  local tries=20
   while (( ${tries} > 0 )) ; do
-    if ${netcat} -z "$1" "$2" 2> /dev/null ; then
+    if rsync "rsync://k8s@${1}:${2}/" \
```

Comment:
There is -- we do 20 tries with a 0.1s sleep between each attempt.  The rsync command line is broken over a couple of lines so that might be confusing.